src/ripple/app/main/Application.cpp

```diff
@@ -844,7 +855,9 @@ class ApplicationImp
 
         mValidations->flush ();
 
-        m_overlay->saveValidatorKeyManifests (getWalletDB ());
+        validators_->stop ();
+
+        manifestCache_->save (getWalletDB ());
```

Comment:
Something like
`//TODO Store manifests in manifests.sqlite instead of wallet.db`?